packages/jest-config/src/normalize.ts

```diff
@@ -1175,6 +1178,21 @@ export default async function normalize(
     newOptions.shard = parseShardPair(argv.shard);
   }
 
+  // Process plugins
+  if (newOptions.plugins && newOptions.plugins.length > 0) {
+    const resolvedPlugins: Array<Config.Plugin> = [];
+    for (const pluginConfig of newOptions.plugins as Array<Config.PluginConfig>) {
```

Comment:
Do we actually need explicit casting `as Array<Config.PluginConfig` here and why?